### ISSUE TITLE
en/corpus-basic.batch apparent regression after recent dict changes

### ISSUE THREAD

**Issue by ampli** [opencog/link-grammar#409]
At the indicated commits, the following sentences that are marked as bad became parseable:

183e47454548531c2119c5350f966043ed8885b7 *As had been green, the party was a big success
183e47454548531c2119c5350f966043ed8885b7 *As had expected the party to be a success, it was a success
183e47454548531 *As had wanted to be expected, the party was a big success
9515e5088186c98a061d753c92c13d5404920e85 *Find someone who wants to do
aca8307f8ad5b1e0aba287336a846fa04a9ad5c5 *He told me that even, his mother likes me
7d7af2ecdc8592b661438be1c58d323f504b1019 *Our program did not run as quickly as said
e14b11656dc795133568e74a01cbfa65081b51da *Ours is more elegant than yours works
aca8307f8ad5b1e0aba287336a846fa04a9ad5c5 *Whatever an outcome, I'm sure he'll still be popular

**Comment by linas** [comment]
Yeah, I know. The older dict handled these sentences correctly, but only because it was making some serious errors in handling sentences with the word "as" in them.  There's no easy fix to get these to not parse, again.
